.eslintrc.yml

```diff
@@ -1,12 +0,0 @@
-env:
```

Comment:
@tumido I'm unclear if this file is needed. You don't have a `.eslintrc.yml` in the probot template, I guess you're good with the defaults for running `eslint`?

Reply:
@durandom TL;DR you don't need it.

ESLint can be configured either by `.eslintrc.*` files or by `eslintConfig` key in `package.json`. (https://eslint.org/docs/latest/user-guide/configuring/) So https://github.com/operate-first/operate-first.github.io/pull/130/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R43-R66 is enough.

Similarly `prettier` can be configured via multiple means, one of which is `prettier` key in `package.json`. (https://prettier.io/docs/en/configuration.html)

I always find it better to have less files in the boilerplate, so in general I prefer to put everything into `package.json` if possible.